### PR TITLE
Kyma 2.9.3

### PR DESCRIPTION
**Pull request by ognyvrac**
**Description**
This PR aims to adopt Kyma 2.9.3, which removes all PodSecurityPolicies and upgrades the version of some monitoring components, and Kubernetes 1.25, which removes PodSecurityPolicy as a whole and is replaced by [Pod Security Standard profiles](https://kubernetes.io/docs/concepts/security/pod-security-standards/). 

Changes proposed in this pull request:
- update CLI version
- remove our PSPs
- introduce the `baseline` Pod Security Standard profile by labeling the namespace
- update tests code to use `batch/v1` as `batch/v1beta1` is deprecated in K8s 1.25

**Pull Request status**

- [x] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: docs/compass/04-01-installation.md [hunk 1]
```diff
@@ -383,9 +383,9 @@ To install the Compass and Runtime components on a single cluster, perform the f
 
 > **NOTE:** During the installation of Kyma, the installed version must match to the one in the [`KYMA_VERSION`](../../installation/resources/KYMA_VERSION) file in the specific Compass commit.
 
-You must have a Kyma installation with an enabled Runtime Agent. For more information, see [Enable Kyma with Runtime Agent](https://github.com/kyma-project/kyma/blob/2.8.4/docs/04-operation-guides/operations/ra-01-enable-kyma-with-runtime-agent.md). Therefore, you must add the compass-runtime-agent module in the compass-system namespace to the list of [minimal kyma components file](../../installation/resources/kyma/kyma-components-minimal.yaml).
+You must have a Kyma installation with an enabled Runtime Agent. For more information, see [Enable Kyma with Runtime Agent](https://github.com/kyma-project/kyma/blob/2.9.3/docs/04-operation-guides/operations/ra-01-enable-kyma-with-runtime-agent.md). Therefore, you must add the compass-runtime-agent module in the compass-system namespace to the list of [minimal kyma components file](../../installation/resources/kyma/kyma-components-minimal.yaml).
```
Comment (ekaterina-mitova) [inline docs/compass/04-01-installation.md:386]: with enabled Runtime Agent

**Review by ekaterina-mitova — DISMISSED** [review]
Looks good to me!

File: chart/compass/charts/prometheus-postgres-exporter/values.yaml [hunk 2]
```diff
@@ -43,11 +43,6 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
-rbac:
-  # Specifies whether RBAC resources should be created
-  create: true
-  # Specifies whether a PodSecurityPolicy should be created
-  pspEnabled: true
```
Comment (PetarTodorovv) [inline chart/compass/charts/prometheus-postgres-exporter/values.yaml:50]: There is also an .md file referring to that value - `compass/chart/compass/charts/prometheus-postgres-exporter/README.md` we can remove it from there as well

File: chart/ory/templates/psp.yaml [hunk 3]
```diff
@@ -1,34 +0,0 @@
-{{- if .Values.maintenance.podSecurityPolicy.enabled }}
```
Comment (PetarTodorovv) [inline chart/ory/templates/psp.yaml:1]: We removed the file but seems we still have the chart configuration `maintenance -> podSecurityPolicy`?

**Review by PetarTodorovv — CHANGES_REQUESTED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Review by ekaterina-mitova — APPROVED** [review]
Looks good to me.

**Pull request merged**